Packs/CheckPointSandBlast/Integrations/CheckPointSandBlast/CheckPointSandBlast.py

```diff
@@ -64,6 +64,11 @@
     'QuotaExpiration',
     'Action',
 ]
+DBOT_SCORE_TRANSLATOR_DICT = {
+    1: "Benign",
```

Comment:
why use these values instead of whats in Common.DBotScore?
We cant use the same logic we have in get_dbotscore ? Or try to unify this logic?

Reply:
We just need the translated version from points to the severity level.

Reply:
lets use to_readable as discussed